src/OverlayRendering.cpp

```diff
@@ -0,0 +1,271 @@
+#include "OverlayRendering.h"
+#include "Prelude.h"
+
+extern uint8_t r_image_font_6_11[2464];
+
+/*
+ * Context for the software renderer.
+ */
+struct bitmap
```

Comment:
I'd prefer if we didn't suddenly stop naming structures as everywhere else in the engine, call it in a more specific way.